### ISSUE TITLE
19.0.5 Cannot Enable CORS

### ISSUE THREAD

**Issue by rhuyn** [SAP/cloud-commerce-spartacus-storefront-docs#244]
This template is to be used for bug reports and small enhancements. For feature requests, please contact the project owner.

Note: For the bug to be accepted, it must be reproducible using the latest release of Spartacus. See our [Contributor Documentation](CONTRIBUTING.md) for more information.

## Environment Details

- Spartacus:  Current
- Browser: Chrome 74
- OS: RHEL 6
- Device: VBlock
- Other environment details: Angular 8

## Steps to Reproduce
-Install 19.0.5 SAP Commerce
-Setup SAP Commerce
-Run Spartacus
-Go to Spartacus (localhost:4200)
## Observed Results

Going to the Spartacus site I get a CORS error:
Access to XMLHttpRequest at 'https://myserver:9002/rest/v2/electronics/cms/pages?fields=DEFAULT&pageType=ContentPage&pageLabelOrId=homepage&lang=en&curr=USD' from origin 'http://myserver' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.

## Expected Results
The electronic store to load

## Offer to Fix
No

## Additional Information
Asked on Slack channel and was told to make a ticket.

**Comment by Xymmer** [comment]
moved this from Release 1.0 project as we didn't make changes, it's pure doc.

**Comment by gladius-mtl** [comment]
Our documentation (https://sap.github.io/cloud-commerce-spartacus-storefront-docs/installing-sap-commerce-cloud/#configuring-cors) is up-to-date with the following settings:

corsfilter.ycommercewebservices.allowedOrigins=http://localhost:4200 https://localhost:4200
corsfilter.ycommercewebservices.allowedMethods=GET HEAD OPTIONS PATCH PUT POST DELETE
corsfilter.ycommercewebservices.allowedHeaders=origin content-type accept authorization cache-control if-none-match

After discussing with @hackergil I can confirm there is no need to mention the corsfilter.acceleratorwebservices.* settings.

**Comment by aksmahajan** [comment]
@gladius-mtl as of yesterday, we needed to update corsfilter.acceleratorservices.allowedOrigins property with value as http://localhost:4200 https://localhost:4200 to make the storefront move ahead from payment page. This was observed after we had already updated all the corsfilter.ycommercewebservices.* settings above.

**Comment by gladius-mtl** [comment]
Hi @aksmahajan 

Thank you for getting in touch!

I have certainly encountered the issue of getting stuck at the payment step before, with the fix being to update `sop.post.url`, as described further down the same documentation page, here: https://sap.github.io/cloud-commerce-spartacus-storefront-docs/installing-sap-commerce-cloud/#possible-issues

Did you already do this step before changing the value of the `corsfilter.acceleratorservices.allowedOrigins ` property?

I'm asking just so I can determine if there is a new issue or not 🙂 

Thank you and kindest regards!

**Comment by aksmahajan** [comment]
Hi @gladius-mtl yes, I had already set the sop.post.url to the value mentioned in the documentation. Made triple sure. Also PFA the screenshot of the setting

<img width="1420" alt="Screen Shot 2019-12-06 at 7 50 14 PM" src="https://user-images.githubusercontent.com/57842343/70365949-0b14de00-1862-11ea-9366-79f23d78e874.png">

Also I recorded a little video to show how I am able to move ahead of payment step as a guest, when corsfilter.acceleratorservices.allowedOrigins was set to allow https://localhost:4200, then after removing it how chrome blocks me with CORS error, and then after re-adding the value to corsfilter.acceleratorservices.allowedOrigins how everything works again.

PFA the video:
[accelerator-services-CORS-issue.mp4.zip](https://github.com/SAP/cloud-commerce-spartacus-storefront-docs/files/3934533/accelerator-services-CORS-issue.mp4.zip)

Please let me know if you need more information.

**Comment by gladius-mtl** [comment]
Hello @aksmahajan !

My apologies for taking so long to reply -- but thank you so much for this excellent information! 
I haven't had a chance to follow up on this topic yet, but I just wanted you to know that I received your message, and I'm very grateful for this information -- and I will definitely be following up on this topic early in the new year.

Thanks again, and kindest regards!
